components/ledger/internal/api/v1/controllers_accounts.go

```diff
@@ -133,48 +130,46 @@ func getAccount(w http.ResponseWriter, r *http.Request) {
 
 	acc, err := l.GetAccountWithVolumes(r.Context(), query)
 	if err != nil {
-		ResponseError(w, r, err)
+		sharedapi.InternalServerError(w, r, err)
```

Comment:
If the account does not exist, should we send a NotFound error instead?

Reply:
Account always exists ;)